src/spotify_to_tidal/__main__.py

```diff
@@ -24,13 +25,21 @@ def main():
         spotify_playlist = spotify_session.playlist(args.uri)
         tidal_playlists = _sync.get_tidal_playlists_dict(tidal_session)
         tidal_playlist = _sync.pick_tidal_playlist_for_spotify_playlist(spotify_playlist, tidal_playlists)
-        _sync.sync_list(spotify_session, tidal_session, [tidal_playlist], config)
+        _sync.sync_playlists_wrapper(spotify_session, tidal_session, [tidal_playlist], config)
+        sync_favorites = args.sync_favorites # only sync favorites if command line argument explicitly passed
+    elif args.sync_favorites:
+        sync_favorites = True # sync only the favorites
     elif config.get('sync_playlists', None):
         # if the config contains a sync_playlists list of mappings then use that
-        _sync.sync_list(spotify_session, tidal_session, _sync.get_playlists_from_config(spotify_session, tidal_session, config), config)
+        _sync.sync_playlists_wrapper(spotify_session, tidal_session, _sync.get_playlists_from_config(spotify_session, tidal_session, config), config)
+        sync_favorites = args.sync_favorites is None and config.get('sync_favorites_default', True)
```

Comment:
I would change the default value of `config.get('sync_favorites_default', True)` to False. Otherwise, every user who already used _spotify to tidal_ before this feature and did not add the `sync_favorites_default: true` entry to the config would also have their favorites synchronized, even if they did not want this.

Reply:
This is intentional, I think most users would want this feature, and if they don't it only takes 2s to disable

Reply:
I understand but honestly this can really mess up existing users' library if they do not intend or want to sync their favorites.

Reply:
Ah yeah I forgot that it erases the existing entries from the user's favorites. I think data loss is unacceptable in this situation tbh, so I'd rather keep the default to True but remove the `_clear_favorites()` functionality entirely. The ordering is much less important here than it is with other playlists

Reply:
Pushed a new commit, please test and let me know if any issues